lib/command/createbucket.php

```diff
@@ -74,7 +74,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 		if ($result) {
 			$output->writeln("Bucket already exists: $bucketName");
 			if (!$input->getOption('update-configuration')) {
-				return;
+				return 1;
```

Comment:
IMO if yu are trying to create a bucket, the bucket already exists, and you did not specify `update-configuration` then you should get an error status (non-zero).

`phan` was telling me that the `return` needs to return some integer. `1` seems good.